### PR TITLE
Replaces map with a fallback shown for users on old ios versions

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

Due to Maplibre not working in < iOS 14.7, we decided to show this overlay as a first step. Ideally we should probably make a video recording of the problem, open an issue on MapLibre and see if we can debug it further - but to move on and get the app out, we're settling on this solution :)

### Preview

Screenshot 2023-01-23 at 15 58 20

### CODE REVIEW

File: web/src/features/map/MapFallback.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const isDevelopmentTools = window.navigator?.vendor === 'Google Inc.';
+
+  // If version is less than 14.7 Maplibre won't work
+  const lowestSupportedVersion = '14_7';
+  console.group('isOldIOSVersion');
+  console.log('isIos', isIos);
+  console.log('!isDevelopmentTools', !isDevelopmentTools);
+  console.log('iosVersion', iosVersion);
+  console.log('iosVersion < lowestSupportedVersio', iosVersion < lowestSupportedVersion);
+  console.groupEnd();
```
Comment (madsnedergaard) [inline web/src/features/map/MapFallback.tsx:17]: We should release this on beta, test it thoroughly on different devices and then remove the console logging here :)

**Comment by VIKTORVAV99** [comment]
Can we mark supported iOS versions for the app store so it don't even show up on older versions?
I realize this is not optimal as iOS 14.7 is not that old and we probably should support it.

An alternative solution might be to get the vite legacy plugin working.

**Comment by tonypls** [comment]
> Can we mark supported iOS versions for the app store so it don't even show up on older versions? I realize this is not optimal as iOS 14.7 is not that old and we probably should support it.
> 
> An alternative solution might be to get the vite legacy plugin working.

I've set the mobile app on the App Store so it's only available to supported versions of iOS 14.7 + but the above is needed for visitors to the website on unsupported iOS versions. 

I think Mads tried with the legacy plugin and it didn't help but it's something that could definitely do with more investigation.

**Comment by VIKTORVAV99** [comment]
> > Can we mark supported iOS versions for the app store so it don't even show up on older versions? I realize this is not optimal as iOS 14.7 is not that old and we probably should support it.
> > An alternative solution might be to get the vite legacy plugin working.
> 
> I've set the mobile app on the App Store so it's only available to supported versions of iOS 14.7 + but the above is needed for visitors to the website on unsupported iOS versions.
> 
> I think Mads tried with the legacy plugin and it didn't help but it's something that could definitely do with more investigation.

Ah then disregard the first part 😅.

I'll do some experimentation with the legacy plugin and terser later today but unfortunately I can't test it on iOS as I have no apple devices...

It would be a net gain and we would support more users either way so it's worth a shot at least, I'll also check how big of a difference it makes to use terser for the bundling size wise (we have to install it for the legacy plugin anyway).

**Comment by tonypls** [comment]
> > > Can we mark supported iOS versions for the app store so it don't even show up on older versions? I realize this is not optimal as iOS 14.7 is not that old and we probably should support it.
> > > An alternative solution might be to get the vite legacy plugin working.
> > 
> > 
> > I've set the mobile app on the App Store so it's only available to supported versions of iOS 14.7 + but the above is needed for visitors to the website on unsupported iOS versions.
> > I think Mads tried with the legacy plugin and it didn't help but it's something that could definitely do with more investigation.
> 
> Ah then disregard the first part 😅.
> 
> I'll do some experimentation with the legacy plugin and terser later today but unfortunately I can't test it on iOS as I have no apple devices...
> 
> It would be a net gain and we would support more users either way so it's worth a shot at least, I'll also check how big of a difference it makes to use terser for the bundling size wise (we have to install it for the legacy plugin anyway).

If you get a branch in a position to test I can run it on my simulator and test iOS device

**Comment by madsnedergaard** [comment]
I've tested and I'm happy with it, should we merge and do a new release (on iOS only?) @tonypls?

**Comment by tonypls** [comment]
The current app will work on unsupported OS on newer devices, i.e. iPhone 11 running iOS 14.5. So I'm not sure we should reduce availability where it's not required. Could we put error catching on the map and show the unsupported screen if error && currently version < supported version?

**Comment by VIKTORVAV99** [comment]
> The current app will work on unsupported OS on newer devices, i.e. iPhone 11 running iOS 14.5. So I'm not sure we should reduce availability where it's not required. Could we put error catching on the map and show the unsupported screen if error && currently version < supported version?

Wait what? There is a hardware requirement blocking the app? If it was just software it should not work no matter the device right?

On a side note Safari is the new IE 11...

**Comment by tonypls** [comment]
> > The current app will work on unsupported OS on newer devices, i.e. iPhone 11 running iOS 14.5. So I'm not sure we should reduce availability where it's not required. Could we put error catching on the map and show the unsupported screen if error && currently version < supported version?
> 
> Wait what? There is a hardware requirement blocking the app? If it was just software it should not work no matter the device right?
> 
> On a side note Safari is the new IE...

Cancel that, I just tested it and you're correct is OS version specific so this PR is good to go!

**Review by tonypls — APPROVED** [review]
Let me know when it's released on beta and I'll run through some tests

**Pull request merged**